src/SelfRoleManager.ts

```diff
@@ -43,48 +46,53 @@ export class SelfRoleManager extends EventEmitter {
   /**
    * Creates an instance of SelfRoleManager.
    * @param {Client} [client] The client that instantiated this Manager
-   * @param {SelfRoleOptions} [options={
+   * @param {SelfRoleOptions} [roleOptions={
```

Comment:
The renaming of this parameter is not part of this PR nor should it be acceptable considering the constructor is part of the public API already available to the users of the library. Thus we should avoid possible breaking changes such as this.